plugins/jabra-file/README.md

```diff
@@ -0,0 +1,35 @@
+# Jabra FILE
+
+## Introduction
+
+This plugin is used to firmware update the Jabra PanaCast50 and
+the Lenovo ThinkSmart Bar 180
+
+## GUID Generation
+
+These devices use the standard USB DeviceInstanceId values, e.g.
+
+* `USB\VID_0B0E&PID_3011`
+
+## Update Behavior
+
+The device is updated at runtime using USB interrupt transfers.
+
+## Vendor ID Security
+
+The vendor ID is set from the USB vendor, in this instance set to `USB:0x0B0E`
+
+## External Interface Access
+
+This plugin requires read/write access to `/dev/bus/usb`.
+
+## Version Considerations
+
+This plugin has been available since fwupd version `x.y.z`.
```

Comment:
2.0.0

Reply:
Would it be possible to get into the 1.9.x version too?

Reply:
Sure, lets get this tidied up and then we can backport.